examples/linux-allchecks.conf

```diff
@@ -126,6 +125,12 @@ arg2="nopasswdlogin"
 [[check.pass]]
 type="GuestDisabledLDM"
 
+[[check]]
+[[check.pass]]
+type="PasswordChanged"
+arg1="934712394827340932-some-hash-here-53298573045238905"
+arg2="cpadmin"
```

Comment:
I'm unsure on this, but is it more semantically intuitive to have arg1 be user and arg2 be hash?